ci/dependabot-pr.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -ex
+
+cd "$(dirname "$0")/.."
+
+if ! echo "$BUILDKITE_BRANCH" | grep -E '^pull/[0-9]+/head$'; then
+  echo "not pull request!?" >&2
+  exit 1
+fi
+
+source ci/rust-version.sh stable
+
+ci/docker-run.sh $rust_nightly_docker_image ci/dependabot-updater.sh
+
+if [[ $(git status --short :**/Cargo.lock | wc -l) -gt 0 ]]; then
```

Comment:
I'd prefer an early return here:
```suggestion
if [[ $(git status --short :**/Cargo.lock | wc -l) -eq 0 ]]; then
  echo --- ok
  exit 0
fi
```
so the bigger iif block can be unindented